Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -722,6 +726,13 @@ public class DependencyResolver<
     ) -> AnySequence<AssignmentSet> {
         var allConstraints = allConstraints
 
+        if enablePrefetching {
+            // Ask all of these containers upfront to do async cloning.
+            for constraint in constraints {
+                provider.getContainer(for: constraint.identifier) { _ in }
```

Comment:
Off topic, but how much more parallelism do we get on large projects if, when we get the container, we do something like try and pick the latest version and also prefetch its dependencies?